charts/wallabag/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+appVersion: 2.4.1
+description: a self hostable application for saving web pages, freely.
+name: wallabag
+version: 1.0.0
+kubeVersion: ">=1.16.0-0"
+keywords:
+- wallabag
+home: https://github.com/k8s-at-home/charts/tree/master/charts/wallabag
+sources:
+- https://github.com/wallabag/docker
+- https://hub.docker.com/wallabag/wallabag
+- https://github.com/k8s-at-home/charts/tree/master/charts/wallabag
```

Comment:
This one is a bit superfluous since it's also the `home` location of the chart

Reply:
Should probably remove it from the template in that case: https://github.com/k8s-at-home/charts/blob/94315f07b2efc206ef79c7a614317b4fca51c61a/templates/chart/Chart.yaml#L13